lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -786,6 +935,7 @@ def _process_logging_relation_changed(self, relation: Relation):
         """
         if self._charm.unit.is_leader():
             relation.data[self._charm.app].update({"loki_push_api": self._loki_push_api})
+            relation.data[self._charm.app].update({"data": self._promtail_binary_url})
```

Comment:
Would it be easy at this point in time to rename `data`?

Reply:
We are discussing the relation data schema in a issue. This part is going to change soon.